gpii/node_modules/preferencesServer/test/preferencesServerTests.js

```diff
@@ -1566,26 +1565,42 @@ gpii.tests.preferencesServer.testMap = [ {
 gpii.tests.preferencesServer.testDefs = fluid.flatten(fluid.transform(gpii.tests.preferencesServer.testMap, function (mapEl) {
     return fluid.transform(mapEl.fixtures, mapEl.build, function (fixture) {
         var common = {
-            config: gpii.tests.preferencesServer.config,
-            pouchTestCaseHolder: "gpii.tests.preferencesServer.pouchTestCaseHolder"
+            config: gpii.tests.preferencesServer.config
         };
 
         return fluid.extend({}, common, fixture);
     });
 }));
 
-gpii.test.bootstrapServer(gpii.tests.preferencesServer.testDefs);
-
-// Run tests for /health endpoint:
-// 1. With a proper database connection: readiness === true
-gpii.test.bootstrapServer(gpii.tests.preferencesServer.health.testDefs);
+fluid.defaults("gpii.tests.preferencesServer.testEnvironment", {
+    gradeNames: ["gpii.test.couchdb.environment.base", "gpii.test.serverEnvironment"],
+    databases: {
+        gpii: {
+            data: [
+                "%gpii-universal/gpii/node_modules/preferencesServer/test/data/gpiiKeys.json",
+                "%gpii-universal/gpii/node_modules/preferencesServer/test/data/prefsSafes.json",
+                "%gpii-universal/testData/dbData/views.json"
+            ]
+        }
+    }
+});
 
-// 2. Without a database connection: readiness === true
-kettle.test.bootstrapServer(gpii.tests.preferencesServer.health.testDefs);
+gpii.tests.preferencesServer.testDefToEnvironment = function (testDef) {
+    return gpii.test.testDefToEnvironment(testDef, "gpii.tests.preferencesServer.testEnvironment", "gpii.test.couchSequenceGrade");
+};
 
-// Run tests for /ready endpoint:
-// 1. With a proper database connection: liveness === true
-gpii.test.bootstrapServer(gpii.tests.preferencesServer.ready.testDefsWithDbConnection);
+gpii.test.runTestDefs(gpii.tests.preferencesServer.testDefs, gpii.tests.preferencesServer.testDefToEnvironment);
 
-// 2. Without a database connection: liveness === false
-kettle.test.bootstrapServer(gpii.tests.preferencesServer.ready.testDefsWithoutDbConnection);
+//// Run tests for /health endpoint:
+//// 1. With a proper database connection: readiness === true
+//gpii.test.runServerTestDefs(gpii.tests.preferencesServer.testDefs);
```

Comment:
Looks like these left commented out from a previous experiment

Reply:
woo, very good catch, I'll confirm that those actually work.

Reply:
Oops, they don't!  Fixing them now.